usecase/usecase-webapi-xwalk-tests/samples/IAPGooglePlay/iap/manifest.json

```diff
@@ -1,12 +1,12 @@
-{
-  "name": "iaptest",
-  "start_url": "index.html",
-  "xwalk_package_id": "org.xwalk.iap",
-  "xwalk_extensions": ["."],
-  "xwalk_android_permissions": [
-    "ACCESS_NETWORK_STATE",
-    "ACCESS_WIFI_STATE",
-    "INTERNET",
-    "IAP"
-  ]
-}
+{
+  "name": "IAP Google Play Demo",
+  "start_url": "iap.html",
+  "xwalk_package_id": "org.crosswalk.iapsample",
+  "xwalk_extensions": ["."],
+  "xwalk_android_permissions": [
+    "ACCESS_NETWORK_STATE",
+    "ACCESS_WIFI_STATE",
+    "INTERNET",
+    "IAP"
```

Comment:
"ACCESS_NETWORK_STATE",  "ACCESS_WIFI_STATE", "INTERNET", these 3 permissions are default granted in app-tools I think(you can confirm this), and for "IAP", which should be "BILLING" with app-tools.